src/Illuminate/Pipeline/Pipeline.php

```diff
@@ -184,11 +184,15 @@ protected function carry()
     /**
      * Parse full pipe string to get name and parameters.
      *
-     * @param  string $pipe
+     * @param  string|array $pipe
      * @return array
      */
-    protected function parsePipeString($pipe)
+    protected function parsePipe($pipe)
```

Comment:
This is a breaking change and as such can go only to `master`.

Reply:
Yeh, we should probably not change the name in 6.x, and rename the method in master.